docs/resources/service_compute.md

```diff
@@ -0,0 +1,621 @@
+---
```

Comment:
why is that though? if I look at `master` I can see https://github.com/fastly/terraform-provider-fastly/blob/master/website/docs/r/service_compute.html.markdown and so that file is effectively named the same. The only difference is the new documentation tool is happy with just `.md` file extension and doesn't require `.html.markdown`.